src/openforms/forms/migrations/0098_fix_forms_conditionals.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.11 on 2024-04-16 12:59
+
+from django.db import migrations
+import re
+
+from openforms.formio.migration_converters import (
+    convert_simple_conditionals,
+)
+
+
+def fix_forms_simple_conditionals(apps, schema_editor):
+    FormDefinition = apps.get_model("forms", "FormDefinition")
+
+    all_definitions = FormDefinition.objects.all()
+    if not len(all_definitions):
+        return
+
+    # Check for simple conditionals where the "when" is not an empty string
+    raw_pattern = r'"conditional": \{[\w\s",:]*"when": "(\w+)"[\w\s",:]*\}'
```

Comment:
this looks like the exact same regex - why don't we define it as a module constant? You can even use `from django.utils.regex_helper import _lazy_re_compile` but I honestly don't expect a big import-time overhead from this.

I'm also not sure about this pattern - the spaces here seem to be an implementation detail, `"conditional":{"when": ...` would still be valid JSON, and the ordering of keys in JSON is undefined, so it feels a bit fragile. Do we even need this regex at all? Since the Formio.js SDK/builder adds all the keys (like `conditional`) anyway, I think we can assume that 90% of the components will have this present and would need to be checked anyway.